Scripts/Modules/MultipleRayInputModule/MultipleRayInputModule.cs

```diff
@@ -102,6 +113,11 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
 			if (m_EventCamera == null)
 				return;
 
+			// World scaling also scales clipping planes
+			var viewerScale = getViewerScale();
+			m_EventCamera.nearClipPlane = m_OriginalNearClipPlane * viewerScale;
```

Comment:
I changed this back to copying the values over from the main camera, but kept it here in the code since we didn't need to do it for every intersection check